content/influxdb/v2.1/notebooks/overview.md

```diff
@@ -10,72 +10,67 @@ menu:
     parent: Notebooks
 ---
 
-Learn about the building blocks of a notebook, how notebooks can help show how data is processed, and discover some common use cases.
+Learn how notebooks can help to streamline and simplify your day-to-day business processes.
 
-To use different notebook cell types, controls, and uses to process data, see:
-- [Notebook concepts](/influxdb/v2.1/notebooks/overview/#notebook-concepts)
-- [Notebook controls](/influxdb/v2.1/notebooks/overview/#notebook-controls)
-- [Notebook cell types](/influxdb/v2.1/notebooks/overview/#notebook-cell-types)
+See an overview of [notebook concepts](/influxdb/v2.1/notebooks/overview/#notebook-concepts), [notebook controls](/influxdb/v2.1/notebooks/overview/#notebook-controls), and [notebook cell types](/influxdb/v2.1/notebooks/overview/#notebook-cell-types) also know as the basic building blocks of a notebook.
 
 ## Notebook concepts
 
-Using notebooks, you can:
+You can think of an InfluxDB notebook as template that contains cells (building blocks) to define how to query, visualize, process, and write data to your buckets. Notebooks help you do the following:
 
-- Create documents with live code, equations, visualizations, and explanatory notes.
-- Create dashboard cells or scheduled tasks.
-- Clean and downsample data.
-- Build runbooks.
-- Document data processing steps.
+- Create snippets of live code, equations, visualizations, and explanatory notes.
+- Create alerts or scheduled tasks.
+- Downsample and normalize data.
+- Build runbooks to share with your teams.
+- Output data to buckets.
 
 ## Notebook controls
 
 The following options appear at the top of each notebook.
 
-### Preview/Run mode
+### Run
 
-- Select **Preview** (or press **Control+Enter**) to display results of each cell without writing data. Helps to verify that cells return expected results before writing data.
-- Select **Run** mode displays results of each cell and writes data to the selected bucket.
+Select {{< caps >}}Run{{< /caps >}} (or press **Control+Enter**) to display results of each cell and write data to the selected bucket.
```

Comment:
Both "Preview" and "Run" modes still exist in OSS. Need to do some oss-/cloud-only magic here.

Reply:
ah, thank you! ...just added note in your PR about running preview...